specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -76,36 +76,29 @@ Legend (terms and table are based on Pragmatic Marketing recommendations):
+
+Since JupyterLab works with notebook files the proposed design should support getting the job steps from them. To the purpose, a new VDK plugin will be introduced which will allow vdk to run steps which came from notebook files.
+
+|       Operation        |                                                                                                                                                                                                                          Flow                                                                                                                                                                                                                          | Covered  use cases | Priority |
+|:----------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------:|:--------:|
+|    Run VDK Jupyter     |                                                                                                                                                                        The user runs a single CLI command which opens the web version of JupyterLab. Examples: vdk jupyter lab                                                                                                                                                                         |         1          |   high   |
+|          Help          |                                                                                                After the user enters JupyterLab he will see the VDK drop down menu where he can find the "Help" option. There he will be able to check for more information about how to use the VDK extension and where to find specific buttons and other UI elements                                                                                                |         1          |   mid    |
+|         Log in         |                                                                                                          After the user enters Jupyter lab he/she will see the VDK drop down menu where he can find the "Log in" option. It is for authentication against Control service. A pop out with the corresponding login options will be introduced.                                                                                                          |         1          |   mid    |
```

Comment:
What's the situation with multiple jupyter instances, is the user session per instance or shared?

Reply:
User session per instance, I think they cannot be shared in the default version but there might be some extensions that allow it to be shared.